bazelbuild/codelabs
# Update all the workspace dependencies

Everything was out of date.
This change mostly does nothing but use the latest of everything.
It also adds the typescript build file to generate_build_files.

Tested by
- ./generate_build_files.sh
- bazel build ...
- bazel test ...
- Follow the instructions at the end of README to run the servers
  - see that it works
- build again with  --incompatible_disable_legacy_proto_provider 

FWIW, this mechanism for creating the WORKSPACE and BUILD files with simple scripts is unsustainable. After you get dependencies updated, you have to go back and comment out just the right lines in various places. If we do more of these, it needs a redesign.
